gatsby-config.js

```diff
@@ -53,6 +54,130 @@ module.exports = {
         extensions: ['js', 'sass', 'scss'],
       },
     },
+
+    /*
+     * Gatsby's data processing layer begins with “source”
```

Comment:
I think It would be better if we remove all the comments from configuration file, because it is very difficult to read. If it has something that developers should know, replace it to README file.